service-api/module/Application/src/Services/MockNinoService.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Services;
+
+use Application\Services\Contract\NINOServiceInterface;
+
+class MockNinoService implements NINOServiceInterface
+{
+    public const NO_MATCH = 'Unable to verify NINO details';
+    public const NOT_ENOUGH_DETAILS = 'Not enough details to continue with this form of identification';
+    public const PASS = 'NINO check complete';
```

Comment:
Just realised these should live on the interface since they're an essential part of the contract.